src/azul/strings.py

```diff
@@ -46,3 +46,22 @@ def pluralize(word: str, count: int) -> str:
         else:
             result += 's'
     return result
+
+
+def collapse_lines(string: str):
```

Comment:
```suggestion
def unwrap(string: str):
```

This will be called a lot so a shorter name makes sense.